### PR TITLE
Trigger workflows on tag pushes

### PR DESCRIPTION
**Pull request by skaramicke**

### CODE REVIEW

File: .github/workflows/publish-android.yml [hunk 1]
```diff
@@ -74,5 +81,5 @@ jobs:
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
           asset_path: ./android/app/build/outputs/apk/release/app-release-unsigned-signed.apk
-          asset_name: swag-app-${{ env.BUILD_VERSION  }}.apk
+          asset_name: swag-app-${{ env.RELEASE_DATE  }}.apk
```
Comment (Imamyable) [inline .github/workflows/publish-android.yml:84]: Not sure about this. I´d like to have the possibility to make two releases on the same date without having to remove the previous one.

Comment (skaramicke) [reply .github/workflows/publish-android.yml:84]: Agreed. The website and backend gets the commit hash appended too. Perhaps this should be converted to reusable and added as a job in publish-docker-images, which should then be renamed.

Comment (Imamyable) [reply .github/workflows/publish-android.yml:84]: sounds like a good idea to me :)

**Comment by skaramicke** [comment]
@Imamyable Ready for re-review

**Review by Imamyable — APPROVED** [review]
lgtm

**Pull request merged**